crates/driver/src/infra/notify/notification.rs

```diff
@@ -66,6 +66,9 @@ pub enum Settlement {
     Revert(TransactionHash),
     /// Transaction started reverting during the submission.
     SimulationRevert,
-    /// Winning solver failed to settle the transaction onchain.
+    /// Transaction was not confirmed in time
+    Expired,
```

Comment:
I find the naming a bit unclear. Maybe `SubmissionDeadlineExceeded` instead?

Reply:
Really? I feel `Settlement::Expired` is pretty clear (and more concise than a word composed of three sub-parts).